src/index.js

```diff
@@ -3,15 +3,15 @@ import {useState, useEffect} from 'react'
 const useLocalStorage = (key, {updateFrequency}) => {
   const [value, setValue] = useState()
 
-  const readFromLocalStorage = () => {
+  const read = () => {
```

Comment:
Why changes these names (same for `write`)?

Reply:
Just for readability. But I reverted these changes in 22b43cf8b3594215f9d53d883466d45c7dceff56